test/models/comment_test.rb

```diff
@@ -5,7 +5,8 @@ class CommentTest < ActiveSupport::TestCase
 
 	test "Comment rating test" do
 		user = FactoryGirl.create(:user)
-		tour = FactoryGirl.create(:tour, :start_location => 'New York', :finish_location => 'Boston')
+    address = FactoryGirl.create(:address)
```

Comment:
My spacing is off here, but you should have the `start_location` and `finish_location` not as strings, but as address models.